ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Context
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+<!--- If you have errors or a stacktrace in the console, here is a good place
+to put it. Please consider using a pastebin service or create a gist.-->
+<!--- Some screenshots are always greatly appreciated here! -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Context
```

Comment:
Or just "merge" them? Unless you really think it's important to have 2 distinct sections.